nms/grafana/GrafanaErrorMessage.js

```diff
@@ -30,7 +30,7 @@ export default function GrafanaErrorMessage(props: {
       <p>
         An unrecoverable error was encountered while trying to load Grafana.
         Please contact your NMS administrator or open an issue on
-        <a href="https://github.com/facebookincubator/magma/issues"> github </a>
+        <a href="https://github.com/magma/magma/issues"> github </a>
```

Comment:
_optional:_ Spelling: We usually use `Github`. The official typing would be `GitHub` I believe.

Reply:
out of scope